initial draft of security contact list (#67)
open-telemetry/sig-security: docs/contact-list.md

# Security Contact List

Beyond the initial fix team, a mailing list is setup to include additional
organizations that can benefit from early CVE access. This mailing list

> is created per repository

Reviewer: ```suggestion
is created per repository.
```

## OpenTelemetry Collector Distro Security Contacts List

Currently on the opentelemetry-collector-distros list are representatives from:

*
*

> ## Membership Criteria

Reviewer 2: Can you please add that membership is decided by the sig-security for the moment? We'll use the criteria below as general guidelines, but we reserve the final decision is on us. 

Once we have practical experience with this process and confidence that the criteria make sense, we can remove ourselves as gatekeepers.

> To be eligible for (opentelemetry-)distros list membership, your distro should:

Author: ```suggestion
The membership is initially decided by sig-security members. To be eligible
for (opentelemetry-)distros list membership, your distro should:
```

* Be an actively maintained Open Source OpenTelemetry distribution

> * Have a userbase not limited to your own organization

Reviewer: ```suggestion
* Have a user base not limited to your own organization
```

* Have a publicly verifiable track record, dating back at least 1 year and
  continuing to present day, of fixing security issues and releasing the 

>   fixes within 10 days of the issues being made public

Reviewer: Would "10 days" be too aggressive? I think depending on the actual severity score, it might be fine to bundle low severity fixes in the next release train, which is normally more than 10 days.

Author: i don't have a strong preference here, this was the timeline suggested by the linux distributions document
